dev/com.ibm.ws.ssl/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -123,6 +123,9 @@
         <AD id="verifyHostname" name="%repertoire.verifyHostname" description="%repertoire.verifyHostname.desc" 
             required="false" type="Boolean" default="false" />
             
+        <AD id="trustCacerts" name="%repertoire.trustCacerts" description="%repertoire.trustCacerts.desc" 
```

Comment:
Is this part of SEC-381? Please introduce new configuration as beta first, do not immediately expose as GA. Add `ibm:beta="true"`

Reply:
Changes have been added in last push.

Reply:
Thank you. I don't understand the rest of the code well enough to know if there are any other behaviour changes that could get introduced before they are ready... I'll leave that to an SME reviewer.